src/Microsoft.OData.Core/ODataMessageWriterSettings.cs

```diff
@@ -181,6 +182,13 @@ public ODataUri ODataUri
         /// </summary>
         public ODataMetadataSelector MetadataSelector { get; set; }
 
+        /// <summary>
+        /// Gets or sets the new line character sequence used when writing multipart messages
+        /// see https://tools.ietf.org/html/rfc2046#section-5.1.1
+        /// A TextWriter uses OS specific newline but rfc2046 requires it to be CRLF.
+        /// </summary>
+        public string MultipartNewLine { get; set; }

```

Comment:
Is it useful to have this as a general setting on the MessageWriterSettings? Do we expect it ever to be used except for the Multipart/Mixed batch case? Is it ever valid to set it to anything other than /r/n for that case?

Are we afraid that, if we change to always writing /r/n for multipart/mixed (as per spec), that we'll break existing batch payloads on LINUX?

Reply:
My thinking is no. there is no reasonable scenario to change it to something else. 
It also is used for all line breaks in RawValueWriter (which handles the multipart messages). Having mixed line breaks woudl require a lot more refactoring and there is no reason at the moment to expect it to break clients..

But I did not want to add this hard coded to the RawValueWriter but "centralize" it on the settings. A compromise might be to make this a constant (or readonly) on MessageWriterSettings.